### PR TITLE
docs: add blog post for ESLint fixers

### PR DESCRIPTION
**Pull request by bradzacher**
Context: https://twitter.com/slicknet/status/1535417827502354432

This is a first draft of a blog post about fixers and their IDE integration.
I'm not entirely sold on the preamble about CLI fixes, but we can work together to figure out what's best!

[📄 View Rendered](https://github.com/bradzacher/new.eslint.org/blob/823662192169ae1083912a93215cd03b600aee7c/src/content/blog/2022-06-XX-ESLint-Fixers.md)

### CODE REVIEW

**Review by nzakas — CHANGES_REQUESTED** [review]
Thanks for this! Just did a quick once over because we haven’t finalized our process or the website yet.

Can you create a top-level folder in the repo called “drafts” and move your file into that? You can also remove the date from the filename, as we will figure out what date to publish on at a later time.

We are working on pushing the new site live so your patience is appreciated!

**Comment by nzakas** [comment]
Please rebase on top of main to get the latest setup. We are now asking posts be saved in src/content/drafts. There is also a README in that directory with more instructions.

**Comment by nzakas** [comment]
@bradzacher just checking in to see if you're still working on this?

**Comment by bradzacher** [comment]
Yes it's still on my todo list!
I've just been short on time lately between work and my infant son.
I think I'll have some time next week to finish this off.

**Comment by nzakas** [comment]
No problem! Take your time.

File: src/content/blog/2022-06-XX-ESLint-Fixers.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+If you want some more granularity you can also [use the `--fix-type` flag](https://eslint.org/docs/user-guide/command-line-interface#--fix-type) to limit the types of rules you want to apply fixers for.
+
+It’s worth noting that using the `--fix` flag has an added bonus in that ESLint will perform multiple runs on your codebase until everything is fixed. This can be helpful in case a fixer causes a different lint problem - ESLint will silently apply fixers until the codebase settles, saving you from having to manually redo your lint run.
+
+
+## Manual Fixes in Your IDE
+
+A more granular approach is to leverage your [IDE integration](https://eslint.org/docs/user-guide/integrations#editors) to selectively apply fixers as you code. Most IDEs will expose a “quick action” or “quick fix” menu which allows you to manually choose to apply the fixer when an problem is reported.
```
Comment (amareshsm) [inline src/content/blog/2022-06-XX-ESLint-Fixers.md:17]:
```suggestion
A more granular approach is to leverage your [IDE integration](https://eslint.org/docs/user-guide/integrations#editors) to selectively apply fixers as you code. Most IDEs will expose a “quick action” or “quick fix” menu which allows you to manually choose to apply the fixer when a problem is reported.
```

**Comment by nzakas** [comment]
Going to close this for now. When you're ready to resubmit, feel free to open a new PR.

**Pull request closed without merging**